lua/doom/modules/langs/utils.lua

```diff
@@ -121,47 +121,50 @@ module.use_mason_package = function(package_name, success_handler, error_handler
   end
   profiler.start("mason|using package " .. package_name)
   local ok, err = xpcall(function()
-    local package = mason.get_package(package_name)
-    if not package:is_installed() then
-      -- If statusline enabled, push the package to the statusline state
-      -- So we can provide feedback to user
-      local statusline = doom.features.statusline
-      if statusline then
-        statusline.state.start_mason_package(package_name)
-      end
-
-      package:install()
-      package:on("install:success", function(handle)
-        -- Remove package from statusline state to hide it
+    local registry = require("mason-registry")
+    registry.refresh(function ()
```

Comment:
See the commit https://github.com/doom-neovim/doom-nvim/pull/432/commits/51412f265ea7c7bad40564fd4fe9a32561f04d57 for more information about why this is needed.